README.md

```diff
@@ -298,6 +299,17 @@ user = client.user 'defunkt'
 [access scopes]: http://developer.github.com/v3/oauth/#scopes
 [app-creds]: http://developer.github.com/v3/#increasing-the-unauthenticated-rate-limit-for-oauth-applications
 
+### GitHub App
+Octokit.rb also supports authentication [using GitHub App](https://docs.github.com/en/developers/apps/managing-github-apps/installing-github-apps). 
+
+[Required generated JWT TOKEN](https://docs.github.com/en/developers/apps/building-github-apps/authenticating-with-github-apps#authenticating-as-a-github-app).
```

Comment:
```suggestion
Octokit.rb also supports authentication [using a GitHub App](https://docs.github.com/en/developers/apps/managing-github-apps/installing-github-apps), which [requires a generated JWT token](https://docs.github.com/en/developers/apps/building-github-apps/authenticating-with-github-apps#authenticating-as-a-github-app).
```

Thanks for contributing! I'd like to make a small change to the wording if that's okay.

Reply:
Thanks! 🙆‍♂️